packages/next/client/future/image.tsx

```diff
@@ -349,6 +350,7 @@ const ImageElement = ({
         className={className}
         // @ts-ignore - TODO: upgrade to `@types/react@17`
         loading={loading}
+        fetchPriority={priority ? "high" : null}
```

Comment:
> IMHO `undefined` should be used as the default value.
> 
> Also, let's add a comment with the `web.dev` link, which explains why `fetchPriority` is set even when `<link rel=preload>` is already used.
> 
> I will also try to deploy a demo, so that we could perform some web vitals benchmarks to see if adding `fetchPriority` really helps.

Ok :)